code/code/misc/spell_info.cc

```diff
@@ -2571,9 +2571,9 @@ void buildSpellArray() {
       LEARN_DO_1, START_DO_NO, LEARN_DO_NO, LEARN_DIFF_UNUSUAL, 0.0, 0, 0);
 
   discArray[SKILL_SAP] = new spellInfo(SKILL_THIEF, DISC_THIEF, DISC_THIEF_FIGHT, 
-    STAT_STR, "sap", TASK_NORMAL, LAG_3, POSITION_STANDING, MANA_0, LIFEFORCE_0, 
-    PRAY_0, 0, SYMBOL_STRESS_0, "", "", "", "", START_20, LEARN_3, START_DO_30, 
-    LEARN_DO_5, START_DO_NO, LEARN_DO_NO, LEARN_DIFF_SKILLS, 0.0, 0, 0);
+    STAT_STR, "sap", TASK_NORMAL, LAG_5, POSITION_STANDING, MANA_0, LIFEFORCE_0, 
+    PRAY_0, 0, SYMBOL_STRESS_0, "", "", "", "", START_20, LEARN_1, START_DO_20, 
+    LEARN_DO_2, START_DO_NO, LEARN_DO_NO, LEARN_DIFF_SKILLS, 0.0, 0, 0);
```

Comment:
```
sap                         Current:  (poor)      Potential:  (very good)

H:478 P:100.0 V:196 E:1,001,530,693 > prac
Basic disciplines:
               Thief Abilities : Learnedness: 100%
```
With START_20 and LEARN_1 you can't fully learn this skill. You'd have to somehow get to 120% learnedness

LEARN_2 means you learn 2% per 1% in the disc so you could do START_50 and LEARN_2 to make it max at 100% for example.